.gitignore

```diff
@@ -62,3 +62,7 @@ packages/python/plotly/jupyterlab_plotly/nbextension/index.js*
 
 test/percy/*.html
 test/percy/pandas2/*.html
+
+# Build artifacts
+packages/python/plotly/plotly/package_data/datasets/amphibians.csv.gz
```

Comment:
same as above - are these files related to the pandas warning? if not, should they be in a separate single-purpose PR?

Reply:
I'll remove this commit from this PR.